crates/notion-core/src/distro/node.rs

```diff
@@ -24,6 +24,7 @@ pub struct NodeDistro {
 
 /// Check if the cached file is valid. It may have been corrupted or interrupted in the middle of
 /// downloading.
+// FIXME(#134) - verify checksum
```

Comment:
Thanks for citing an issue! Let's use consistent formatting and write `ISSUE(#134)` instead of `FIXME`.

Reply:
done